maps/config/horizon.dm

```diff
@@ -1,5 +1,5 @@
 // #ifdef XMAS
-// #include "..\warwip\atlas_horizon_xmas_2019.dmm"
+// #include "..\horizon_xmas.dmm"
```

Comment:
you'll need to un comment the includes

Reply:
Got it! Thank you for your patience on this PR!